packages/sit-onyx/src/components/OnyxButton/OnyxButton.stories.ts

```diff
@@ -0,0 +1,52 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxButton from "./OnyxButton.vue";
+
+/**
+ * Buttons serve as fundamental components in UI design,
+ * acting as gateways for user interactions and pivotal points for
+ * initiating actions within an interface.
+ * Whether prompting users to submit forms,
+ * navigate through pages, or trigger specific functionalities,
+ * buttons play a pivotal role in guiding users through their
+ * digital journey.
+ */
+const meta: Meta<typeof OnyxButton> = {
+  title: "components/OnyxButton",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxButton,
+    events: ["click"],
```

Comment:
We need to define the options for the props that only have specific values to select from so Storybook displays a dropdown.
![image](https://github.com/SchwarzIT/onyx/assets/67898185/b19b7345-eab9-49ca-9853-79ddf74718a5)

```suggestion
    events: ["click"],
    argTypes: {
      type: {
        options: ["a", "b"],
      },
    },
```

I'd recommend to create a `types.ts` file where you can put the types so get an array for Storybook as well as the type to use in the props:

```ts
export const BUTTON_TYPES = ["button", "submit", "reset"] as const;
export type ButtonType = (typeof BUTTON_TYPES)[number];
```